src/SmartComponents/SystemInventory/SystemInventoryHeader.js

```diff
@@ -144,7 +144,17 @@ const SystemInventoryHeader = ({
                                 <FailState appName='Inventory' isSmall />
                             }
                         </Flex>
-                        <FlexItem align={{ md: 'alignRight' }}>
+                        <FlexItem align={{ md: 'alignRight' }} margin-right={{}}>
```

Comment:
Hmmm looks like you forgot to remove `margin-right={{}}` here :smile:

Reply:
ahh I meant to remove that, updating it now